antarest/study/service.py

```diff
@@ -206,6 +224,54 @@ def assert_raw(study: Study) -> RawStudy:
     return study
 
 
+class StudySettingsDTO(AntaresBaseModel):
+    time_series: TimeSeriesConfiguration
+    general: GeneralConfig
+    advanced_parameters: AdvancedParameters
+    adequacy_patch: AdequacyPatchParameters
+    optimization: OptimizationPreferences
+    thematic_trimming: ThematicTrimming
+    playlist: Playlist
+
+
+class StudyXpansionDTO(AntaresBaseModel):
+    settings: XpansionSettings
+    candidates: list[XpansionCandidate]
+    constraint: bytes
+
+
+class StudyShortTermStorageDTO(AntaresBaseModel):
+    cluster: STStorage
```

Comment:
We should not name it cluster, it's not a cluster.
In any case, it would be better to have something like that with the name and id and other properties at the first level:
```json
{
  "id": "...",
  "name": "...",
  "injectionNominalCapacity": 100,
  ...,
  "constraints": [

  ]
}
```